.github/workflows/build_changed_recipes.yaml

```diff
@@ -87,28 +84,23 @@ jobs:
         shell: bash -l {0}
         run: |
           micromamba activate ci-env
-          python -m pip install git+https://github.com/emscripten-forge/emboa@main
+          python -m pip install emperator
           python -m pip install git+https://github.com/DerThorsten/boa.git@update
           python -m pip install git+https://github.com/DerThorsten/bitfurnace.git@emscripten_new
           python -m pip install playwright
           playwright install
 
-
-
-
       ################################################################
       # build recipes with changes PUSH
       ################################################################
       - name: "build packages push"
         shell: bash -l {0}
         if: github.event_name == 'push'
         run: |
+          micromamba activate ci-env
           cd ${GITHUB_WORKSPACE}
-          export CONDA_EMSDK_DIR=$HOME/src/emsdk
-          pushd $CONDA_EMSDK_DIR
-          ./emsdk activate ${{matrix.emsdk_ver}}
-          source emsdk_env.sh
-          export FILE_PACKAGER=$CONDA_EMSDK_DIR/upstream/emscripten/tools/file_packager.py
```

Comment:
We shouldn't need this FILE_PACKAGE anymore since https://github.com/emscripten-forge/emperator/pull/4